src/write.rs

```diff
@@ -0,0 +1,30 @@
+use std::fmt;
+use Event;
+
+#[derive(Debug)]
+pub struct Error;
+
+impl From<fmt::Error> for Error {
+    fn from(_: fmt::Error) -> Error {
+        Error
+    }
+}
+
+pub trait WriteHtml {
+    fn write_event(&mut self, event: Event) -> Result<(), Error>;
+    fn write_events<'a, I>(&mut self, events: I) -> Result<(), Error>
```

Comment:
Since `Event` now implements `Display` I was having a look at what it's like not using `WriteHtml` at all. I think it can probably be deleted.